HouseRules_Essentials/README.md

```diff
@@ -263,26 +293,30 @@ The [Settings Reference](../docs/SettingsReference.md) contains lists of all dif
     }
   },
   ```
+  </default>
```

Comment:
Shouldn't this be a </default> tag? Currently wen you click the 'show json' tag for this, it mashes up the results.